hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/DBCheckpointServlet.java

```diff
@@ -64,10 +67,14 @@
   private transient DBCheckpointMetrics dbMetrics;
 
   private boolean aclEnabled;
+  private boolean isSpnegoEnabled;
   private Collection<String> ozAdmins;
+  private String reconSPN;
 
+  @SuppressWarnings("checkstyle:hiddenfield")
   public void initialize(DBStore store, DBCheckpointMetrics metrics,
-      boolean omAclEnabled, Collection<String> ozoneAdmins)
+                         boolean omAclEnabled, Collection<String> ozoneAdmins,
```

Comment:
Can we refactor the initialize() with just a Collection<String> allowedUsers? and let the callers to combine ozoneAdmin and recon SPN without additional checkstyle warning suppress?